Nemo Connect/Sources/NCWebServiceParameterParser.m

```diff
@@ -16,7 +16,7 @@ - (NSString *)parseServiceQueryString:(NSString *)serviceRootQuery parameters:(N
 {
     NSMutableString *mutableServiceRootQuery = [serviceRootQuery mutableCopy];
 
-    NSString *pattern = @"%\\d\\[.+\\]|%\\d";
+    NSString *pattern = @"%\\d\\[\\w+\\]|%\\d";
```

Comment:
This was changed because the old patter failed in the situation when we have more than 1 parameter. so for example if the request contains this: /path?parameter=object[property1]&parameter2=object[property2]. The original patter recognise this string: [property1]&parameter2=object[property2] because it is looking for a '[' then any number of any char ('.+') and then ']'. The new pattern is fixing this problem.